Review edits (#58)
dib-lab/2020-workflows-paper: content/05.project-management.md

Project management, the strategies and decisions used to keep a project organized, documented, functional, and shareable, is foundational to any research program.
Clear organization and management is a learned skill that takes time to implement.

> Workflow systems both simplify and improve computational project management, but even workflows that are fully specified in workflow systems require additional investment to stay organized, documented, and backed up.

Reviewer: I think: omit 'both'.

> Workflow systems both simplify and improve computational project management, but even workflows that are fully specified in workflow systems require additional investment to stay organized, documented, and backed up.

Author: ```suggestion
Workflow systems simplify and improve computational project management, but even workflows that are fully specified in workflow systems require additional investment to stay organized, documented, and backed up.
```

![A directed acyclic graph (DAG) that illustrates connections between all steps of a sequencing data analysis workflow. Each box represents a step in the workflow, while lines connect sequential steps.
The DAG shown in this figure illustrates a real bioinformatics workflow for RNA-seq quantification was generated by modifying the default Snakemake workflow DAG.

> This example of an initial workflow used only to quality control and then quantify one fastq file against a transcriptome more than doubles the amount of files in a project. 

Reviewer: fastq should be FASTQ, right? (Not actually sure, but googling suggests.)

> This example of an initial workflow used only to quality control and then quantify one fastq file against a transcriptome more than doubles the amount of files in a project. 

Author: ```suggestion
This example of an initial workflow used only to quality control and then quantify one FASTQ file against a transcriptome more than doubles the amount of files in a project. 
```

![A directed acyclic graph (DAG) that illustrates connections between all steps of a sequencing data analysis workflow. Each box represents a step in the workflow, while lines connect sequential steps.
The DAG shown in this figure illustrates a real bioinformatics workflow for RNA-seq quantification was generated by modifying the default Snakemake workflow DAG.

> With many input files, even this straightforward workflow can result in hundreds of intermediate files. Fortunately, workflow system coordination alleviates the need for a user to directly manage file interdependencies.

Author: ```suggestion
This example of an initial workflow used only to quality control and then quantify one fastq file against a transcriptome more than doubles the amount of files in a project. 
When the number of steps are expanded to carry out a full research analysis and the number of initial input files are increased, a workflow can generate hundreds to thousands of intermediate files.
Fortunately, workflow system coordination alleviates the need for a user to directly manage file interdependencies.
```

Author: thanks for suggestion @shannonekj - look ok?

Reviewer 2: looks great @bluegenes!

Author: thanks!
